uui/services/__tests__/ModalContext.test.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { ModalContext } from "../ModalContext";
+import { LayoutContext } from "../LayoutContext";
+
+describe("ModalContext", () => {
+    it("should create operations", () => {
+        const context = new ModalContext(new LayoutContext());
+
+        const render = () => <div/>;
+
+        context.show(render);
```

Comment:
test with witch arguments operation created and test success and abort callback functionality also